XLForm/XL/XLFormDateCell.m

```diff
@@ -73,7 +75,8 @@ -(BOOL)becomeFirstResponder
             XLFormRowDescriptor * datePickerRowDescriptor = [XLFormRowDescriptor formRowDescriptorWithTag:nil rowType:XLFormRowDescriptorTypeDatePicker];
             XLFormDatePickerCell * datePickerCell = (XLFormDatePickerCell *)[datePickerRowDescriptor cellForFormController:self.formViewController];
             [self setModeToDatePicker:datePickerCell.datePicker];
-            if (self.rowDescriptor.value){
+            [self setStyleToDatePicker:datePickerCell.datePicker];
```

Comment:
Looks like we lose the default date logic here? This is what we have in the forked version

```obj-c
            [self setStyleToDatePicker:datePickerCell.datePicker];
            if (self.rowDescriptor.value){                
                [datePickerCell.datePicker setDate:self.rowDescriptor.value animated:[self.rowDescriptor.rowType isEqualToString:XLFormRowDescriptorTypeCountDownTimerInline]];
            } else if (self.defaultDate) {
                [datePickerCell.datePicker setDate:self.defaultDate];
            }
```

Reply:
It is still there @nkennedyKit if you open the file. 

Only date new style is added here 
`[self setStyleToDatePicker:datePickerCell.datePicker];`

Complete code looks like this 
```
[self setModeToDatePicker:datePickerCell.datePicker];
[self setStyleToDatePicker:datePickerCell.datePicker]; <----- New code
if (self.rowDescriptor.value){
    [datePickerCell.datePicker setDate:self.rowDescriptor.value animated:[self.rowDescriptor.rowType isEqualToString:XLFormRowDescriptorTypeCountDownTimerInline]];
} else if (self.defaultDate) {
    [datePickerCell.datePicker setDate:self.defaultDate];
}

```